### ISSUE TITLE
Yu et al 2022

### ISSUE THREAD

**Issue by standage** [bioforensics/MicroHapDB#102]
## Paper

*If available, please provide a citation for a paper or preprint describing the markers and/or allele frequencies.*

Yu WS, Feng YS, Kang KL, Zhang C, Ji AQ, Ye J, Wang L (2022) Screening of highly discriminative microhaplotype markers for individual identification and mixture deconvolution in East Asian populations. *Forensic Science International: Genetics*, 59:102720, [doi:10.1016/j.fsigen.2022.102720](https://doi.org/10.1016/j.fsigen.2022.102720).

## Data URL(s)

*Please provide URL(s) from which the relevant data file(s) can be downloaded.*

https://doi.org/10.1016/j.fsigen.2022.102720

[mmc1.txt](https://github.com/bioforensics/MicroHapDB/files/10144300/mmc1.txt)

[mmc1.xlsx](https://github.com/bioforensics/MicroHapDB/files/10144301/mmc1.xlsx)

## Author and PI information

Le Wang wangle_02@163.com

**Comment by standage** [comment]
```
>>> import microhapdb
>>> notyu = microhapdb.markers[microhapdb.markers.Source != "Yu2022"]
>>> yu = microhapdb.markers[microhapdb.markers.Source == "Yu2022"]
>>> notyu_rsids = set()
>>> for i, row in notyu.iterrows():
...   notyu_rsids.update(row.RSIDs.split(";"))
... 
>>> len(notyu_rsids)
3087
>>> yu_overlap = set()
>>> for i, row in yu.iterrows():
...   for rsid in row.RSIDs.split(";"):
...     if rsid in notyu_rsids:
...       yu_overlap.add(row.Name)
... 
>>> len(yu_overlap)
42
>>> yu_overlap
{'mh02WL-035', 'mh14WL-009', 'mh22WL-010', 'mh15WL-043', 'mh15WL-003', 'mh01WL-057', 'mh02WL-036', 'mh05WL-019', 'mh09WL-006', 'mh19WL-009', 'mh03WL-022', 'mh01WL-096', 'mh08WL-026', 'mh02WL-029', 'mh04WL-006', 'mh12WL-016', 'mh02WL-103', 'mh04WL-007', 'mh09WL-004', 'mh09WL-036', 'mh13WL-010', 'mh16WL-004', 'mh04WL-005', 'mh13WL-042', 'mh13WL-040', 'mh01WL-012', 'mh01WL-043', 'mh17WL-029', 'mh19WL-001', 'mh09WL-003', 'mh19WL-023', 'mh01WL-001', 'mh05WL-010', 'mh10WL-047', 'mh11WL-010', 'mh21WL-004', 'mh03WL-090', 'mh02WL-078', 'mh10WL-013', 'mh14WL-001', 'mh05WL-063', 'mh03WL-001'}
```

**Comment by standage** [comment]
It was while preparing this data set for integration that I recognized the need for merging microhap IDs that overlap on the genome. This was recently implemented in #108. Of the 703 markers defined in Yu 2022, 44(?) overlap with microhaps defined previously with different names, while 659 seem to be novel.

```
$ grep Yu2022 marker.csv | cut -f 1 -d , | cut -c 5- | cut -f 1 -d '-' | sort | uniq -c
      6 FHL
     11 KK
      1 LV
     18 SHY
      5 USC
    659 WL
      1 ZBF
      2 ZHA
```
